docs/source/example_utils.py

```diff
@@ -21,13 +20,14 @@ def plot_image(image: SimpleITK.Image):
     plt.imshow(nda, cmap='gray')
 
 
-def run_gaussian():
+def run_example(module_name, func_name, args):
+    sys.path.append(f'../../Examples/' + module_name)
```

Comment:
Please document this functions and arguments with doc strings: https://peps.python.org/pep-0257/

Reply:
You might as well use a standard as oppose to something arbitrary.